### PR TITLE
Update Cyberduck FAQ

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

File: cyberduck/faq.md [hunk 1]
```diff
@@ -127,6 +127,16 @@ See _[Preferences → Update](preferences.md#update)_ to switch the update feed.
 - Sophos Anti-Virus *On-access Scanning*
 - Opening the web browser to authenticate using OAuth does not work with [Browser Chooser 2](https://browserchooser2.com/).
 
-## Why does Cyberduck autofil the passwords?
+## Use cases
 
-Password will be autofilled as long as you choose to enable the _Use Keychain_ option in the Cyberduck preferences [Connection tab](connection.md#passwords).
\ No newline at end of file
+### Why does Cyberduck autofil the passwords?
```
Comment (dkocher) [inline cyberduck/faq.md:132]:
```suggestion
### Why does Cyberduck autofilf the passwords?
```

File: cyberduck/faq.md [hunk 2]
```diff
@@ -127,6 +127,16 @@ See _[Preferences → Update](preferences.md#update)_ to switch the update feed.
...
+
+Password will be autofilled as long as you choose to enable the _Use Keychain_ option in the Cyberduck preferences [Connection tab](connection.md#passwords).
+
+### Why do I always get a warning sign when I try to set up a bookmark connection?
```
Comment (dkocher) [inline cyberduck/faq.md:136]:
```suggestion
### Why do I always get a warning sign when I try to set up a connection?
```

File: cyberduck/faq.md [hunk 3]
```diff
@@ -127,6 +127,16 @@ See _[Preferences → Update](preferences.md#update)_ to switch the update feed.
...
+
+A warning sign next to the URL indicates that the URL may not be accessible due to a missing hostname in the _Server_ field or an invalid combination of hostname, credentials, and path.
```
Comment (dkocher) [inline cyberduck/faq.md:138]:
```suggestion
A warning sign next to the URL indicates that the URL may not be reachable due to a missing or invalid hostname in the _Server_ field.
```

File: cyberduck/faq.md [hunk 4]
```diff
@@ -127,6 +127,16 @@ See _[Preferences → Update](preferences.md#update)_ to switch the update feed.
 - Sophos Anti-Virus *On-access Scanning*
 - Opening the web browser to authenticate using OAuth does not work with [Browser Chooser 2](https://browserchooser2.com/).
 
-## Why does Cyberduck autofil the passwords?
+## Use cases
```
Comment (dkocher) [inline cyberduck/faq.md:130]: I would keep the FAQ short and move this documentation to specific pages such as 

- [ ] https://docs.cyberduck.io/cyberduck/connection/

**Review by dkocher — CHANGES_REQUESTED** [review]

File: cyberduck/faq.md (hunk 4 above)
Comment (Cassie205) [reply cyberduck/faq.md]: Content was moved to https://docs.cyberduck.io/cyberduck/connection/#problems-connecting

**Review by dkocher — APPROVED** [review]

**Pull request merged**